src/ImpactX.cpp

```diff
@@ -117,4 +117,18 @@ namespace impactx
         } // end step loop
     }
 
+    void ImpactX::initElements ()
+    {
+        // initialize element sequence
+        myelements = { };
+
+        // add elements
+        Drift drift1(1.0);
+        Drift drift2(0.5);
+        myelements.push_back(drift1);
+        myelements.push_back(drift2);
```

Comment:
We can even make it more compact and avoid a copy by using:
```suggestion
        myelements.emplace_back(Drift(1.0));
        myelements.emplace_back(Drift(0.5));
```